src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts

```diff
@@ -169,6 +166,32 @@ export const thorchainApi: SwapperApi = {
         }
       }
       case TradeType.L1ToLongTail:
+        assert(router, 'router required for l1 to thorchain longtail swaps')
+
+        const publicClient = viemClientByChainId[chainId as EvmChainId]
+        assert(publicClient !== undefined, `no public client found for chainId '${chainId}'`)
```

Comment:
I can see this was copied over from the previous implementation but `publicClient` isn't anywhere here, is there any rationale for the check @0xApotheosis ?

Reply:
That's a good question. I suspect we removed whatever was consuming it at some point in the original implementation but never caught the instantiation due to the assertion immediately below it.

I suspect this instance and the original instance can both be removed.

Reply:
Kicked it out, nice catch